api/services/likeService.js

```diff
@@ -0,0 +1,24 @@
+const { likeDao } = require('../models');
+const { CustomError } = require('../utils/error');
+
+const createLike = async (userId, productId) => {
+    const checkLike = await likeDao.getLikeUserId(userId, productId);
+
+    if (checkLike?.prodcutId) {
```

Comment:
```suggestion
    if (checkLike) {
```
이렇게 해도 의도하시는 동작과 똑같지 않나요?